### DISCUSSION TITLE
Short Spanish words not understood correctly.  Nova-2 multi-language.

### DISCUSSION THREAD

**Discussion by jeremyvillalobos** [deepgram/community#1027] [General help]
It is very easy to reproduce this error were short Spanish words like "Alo", "Si" and "Bien" are no heard at all by the engine or understood as English words.

The word 'Si' is often not heard at all. Even if I pair with more words it is not registering. Like "si senor" will be understood as senor.
Bien and alo are often times interpreted as Yeah 

"uuid": "62769322-d9dc-4ad8-b83b-084051b9d7ab"
"uuid": "7c67b1e3-be51-4337-bca8-ce18de095635"

**Comment by deepgram-community** [comment]
Thanks for asking your question. Please be sure to reply with as much detail as possible so the community can assist you efficiently.
<sub>Consider joining our [Discord community](https://discord.gg/deepgram) for more opportunity to engage with your fellow Deepgram users. You can earn points which can be redeemed for cool stuff by being active in our communities!</sub>

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Reply by slavikshen** [reply]
> Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

Hi, this issue is in project/6ab18422-7cea-4b09-98ea-866a7ca04ea2

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- The deepgram product you are using (e.g Speech to Text, Agent API)

**Comment by deepgram-community** [comment]
Hello, I noticed you are using the `multi` param in your request which will use our English / Spanish Multi-lingual model.

Do you have both English and Spanish audio to transcribe in the same source?  If you just have Spanish audio you'll likely get better results just using the Spanish model for nova-2

so instead of:

```{
"model":"nova-2"
"language":"multi"
"punctuate":"true"
"endpointing":"100"
"encoding":"mulaw"
"sample_rate":"8000"
}```
try using:

```{
"model":"nova-2"
"language":"es"
"punctuate":"true"
"endpointing":"100"
"encoding":"mulaw"
"sample_rate":"8000"
}```
Here are the Docs on Nova-2's language options:

<https://developers.deepgram.com/docs/models-languages-overview#nova-2>

> ![avatar](https://avatars.slack-edge.com/2023-02-20/4855338233600_fad9174023ee3f640ce1_32.png) This message was sent by John Vajda from Deepgram, via our community automation.

**Reply by slavikshen** [reply]
We need the model to understand both English & Spanish in the same stream.  The callee may switch the language at any time.

**Comment by deepgram-community** [comment]
Understood. This is one of our newer models and it still being improved in terms of language detection as it detects multiple languages.

 The quality of the audio, background noise and type of audio you are passing us (mp3, flac, acc etc) can all play a part in the transcription process as well.


You could trying boosting words that are being missed with our Keywords feature: <https://developers.deepgram.com/docs/keywords>

Smart Formatting can also help in some cases:
<https://developers.deepgram.com/docs/smart-format>

But as for the models performance on these common Spanish words I can share this feedback with our Model Research Team.

> ![avatar](https://avatars.slack-edge.com/2023-02-20/4855338233600_fad9174023ee3f640ce1_32.png) This message was sent by John Vajda from Deepgram, via our community automation.
